src/vs/workbench/contrib/accessibility/browser/accessibilityConfiguration.ts

```diff
@@ -71,7 +71,7 @@ const configuration: IConfigurationNode = {
 			...baseProperty
 		},
 		[AccessibilitySettingId.UnfocusedViewOpacity]: {
-			description: localize('unfocusedViewOpacity', 'The opacity percentage (0.2 to 1.0) to use for unfocused editors and terminals.'),
+			description: localize('unfocusedViewOpacity', 'The opacity fraction (0.2 to 1.0) to use for unfocused editors and terminals. This will dim inactive views to make the focused views more obvious.'),
```

Comment:
> "make the focused views more obvious"

@Tyriar would "view" be more accurate here? Can more than one view be simultaneously focused?

Reply:
:+1: https://github.com/microsoft/vscode/pull/190740